<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyChatter Server</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>send.svg</normaloff>send.svg</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>586</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="main_layout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="lbl">
        <property name="minimumSize">
         <size>
          <width>441</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Symbols Nerd Font</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid white;</string>
        </property>
        <property name="text">
         <string>Chatter Server</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="status"/>
      </item>
      <item>
       <widget class="QScrollArea" name="scroll">
        <property name="minimumSize">
         <size>
          <width>439</width>
          <height>419</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignJustify|Qt::AlignTop</set>
        </property>
        <widget class="QWidget" name="scroll_content">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>484</width>
           <height>413</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <item row="1" column="0">
         <widget class="QTextEdit" name="in_txt">
          <property name="minimumSize">
           <size>
            <width>348</width>
            <height>69</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="enc_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>70</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>100</width>
            <height>200</height>
           </size>
          </property>
          <property name="text">
           <string>Send!</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHey">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="setAddr"/>
   </widget>
   <addaction name="menuHey"/>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="setAddr">
   <property name="text">
    <string>Server Address</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
